docs/reference/mapping/types/parent-join.asciidoc

```diff
@@ -217,14 +199,6 @@ Will return:
                         "parent": "1"
                     }
                 },
-                "fields": {
-                    "my_join_field": [
-                        "my_child"
-                    ],
-                    "my_join_field#my_parent": [
-                        "1"
-                    ]
-                },
                 "sort": [
```

Comment:
The same change is needed in `reference/search/request/inner-hits` since we also check the output in this doc.